geomesa-arrow/geomesa-arrow-tools/bin/geomesa-arrow

```diff
@@ -47,6 +47,14 @@ else
       GEOMESA_OPTS="$GEOMESA_OPTS $GEOMESA_DEBUG_OPTS"
       shift 1
     fi
-    java ${CUSTOM_JAVA_OPTS} ${GEOMESA_OPTS} -cp ${CLASSPATH} org.locationtech.geomesa.arrow.tools.ArrowRunner "$@"
+    cmd="java ${CUSTOM_JAVA_OPTS} ${GEOMESA_OPTS} -cp ${CLASSPATH} org.locationtech.geomesa.arrow.tools.ArrowRunner ${@}"
+    if isExportLeaflet $@; then
+      exec 3>&1 # Create a new output file to redirect sub-shell's output here
+      res=$(eval $cmd 2>&1 | tee >(cat - >&3); exit ${PIPESTATUS[0]})
```

Comment:
hmm, i'm still not crazy about all this bash-fu to launch the browser - @jahhulbert-ccri what do you think?

Reply:
i don't think we should launch it by default

Reply:
i prefer not to have our tools launch any other programs since that's not "unix-y"